images/scripts/rhel.setup

```diff
@@ -34,8 +34,39 @@ set -x
 
 if [ ! -f "$SKIP_REPO_FLAG" ]; then
     # Configure repositories.
+    if [[ "$SERVER_REPO_URL" && "$EXTRAS_REPO_URL" ]]; then
+cat <<EOF > /etc/yum.repos.d/rel-eng.repo
+[RHEL-7]
+name=rhel7-server
+baseurl=$SERVER_REPO_URL
+enabled=1
+gpgcheck=0
+
+[EXTRAS-7]
+name=rhel7-extras
+baseurl=$EXTRAS_REPO_URL
+enabled=1
+gpgcheck=0
+EOF
+        $YUM_INSTALL yum-utils
```

Comment:
The "regular" rhel-7 cases do the `rm -f --verbose /etc/yum.repos.d/*.repo` thing, to avoid having multiple repos that might collide with the specified compose. I think you want this as well.

Reply:
Good point, fixed :). Thanks.